ansible/ansible_collections/agentic/agent_configs/tests/unit/plugins/modules/test_packaging_modules.py

```
@@ -0,0 +1,447 @@
+    assert_equal(
+        resolver(explicit_param),
+        str(Path(home) / expected_suffix),
+        f"{resolver.__name__} should prefer explicit value, then {env_var_name}",
+    )
+
+
+def test_bun_global_check_mode_installs_missing_package(
+    tmp_path: Path,
+    monkeypatch: pytest.MonkeyPatch,
+) -> None:
+    monkeypatch.setattr(bun_global, "resolve_binary", lambda module, value: "/usr/bin/bun")
```

Ruff (0.15.11)

[warning] Unused lambda argument: `module`

(ARG005)